doc/reference/hv-make-options.rst

```diff
@@ -0,0 +1,199 @@
+.. _hypervisor-make-options:
+
+Hypervisor Makefile Options
+###########################
+
+The ACRN hypervisor source code provides a makefile to build the ACRN
+hypervisor binary and associated components.
+
+Assuming that you are at the top level of the ``acrn-hypervisor`` directory,
+you can run the ``make`` command to start the build. See :ref:`acrn_configuration_tool` for information about required input files.
+
+Build Options
+*************
+
+The following table shows ACRN-specific command-line options:
+
+.. list-table::
+   :widths: 33 77
+   :header-rows: 1
+
+   * - Option
+     - Description
+
+   * - ``BOARD``
+     - Required. Path to the board configuration file.
+
+   * - ``SCENARIO``
+     - Required. Path to the scenario configuration file.
+
+   * - ``RELEASE``
+     - Optional. Build a release version or a debug version. Valid values
+       are ``y`` for release version or ``n`` for debug version. (Default
+       is ``n``.)
+
+   * - ``hypervisor``
+     - Optional. Build the hypervisor only.
+
+   * - ``devicemodel``
+     - Optional. Build the Device Model only.
```

Comment:
Actually, the `devicemodel` target depends on `tools`... so even if you only want to build `devicemodel`, you are also automatically building tools.

Reply:
@gvancuts Can you suggest edits to the text/table?

Reply:
> Actually, the `devicemodel` target depends on `tools`... so even if you only want to build `devicemodel`, you are also automatically building tools.

@gvancuts Should I combine devicemodel and tools as follows:

   * - ``devicemodel tools``
     - Optional. Build the Device Model and tools only.

Reply:
I'm thinking we can remove the "only" word in all three descriptions so they read: `Build the XXX.`. In the case of the `devicemodel`, I would simply add a note that the `tools` will also be built as a dependency.